pkg/gcv/configs/config_test.go

```diff
@@ -211,3 +212,24 @@ func TestFixLegacyMatcher(t *testing.T) {
 		})
 	}
 }
+
+func TestNewConfiguration(t *testing.T) {
+	config, err := NewConfiguration([]string{"../../../test/cf"}, "../../../test/cf/library")
+	if err != nil {
+		t.Fatalf("want no error, got %s", err)
+	}
+
+	// Check the length of the slices
+	configVal := reflect.ValueOf(config).Elem()
```

Comment:
why do we need to use reflection here? is it to save a few comparison statements?

Reply:
yes, only reason, changed back.